src/test/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineViewTest/testMyUserIdCauseConversionNoStatic/jobs/static-job/builds/2014-11-06_11-46-33/build.xml

```diff
@@ -0,0 +1,200 @@
+              </markupFormatter>
+              <jdks/>
+              <viewsTabBar class="hudson.views.DefaultViewsTabBar"/>
+              <myViewsTabBar class="hudson.views.DefaultMyViewsTabBar"/>
+              <clouds/>
+              <slaves></slaves>
+              <quietPeriod>5</quietPeriod>
+              <scmCheckoutRetryCount>0</scmCheckoutRetryCount>
+              <views>
+                <hudson.model.AllView>
+                  <owner class="hudson.model.Hudson" reference="../../.."/>
+                  <name>All</name>
+                  <description>description</description>
+                  <filterExecutors>false</filterExecutors>
+                  <filterQueue>false</filterQueue>
+                  <properties class="hudson.model.View$PropertyList"/>
+                </hudson.model.AllView>
+              </views>
+              <primaryView>All</primaryView>
+              <slaveAgentPort>15000</slaveAgentPort>
+              <label></label>
+              <nodeProperties/>
+              <globalNodeProperties></globalNodeProperties>
+              <noUsageStatistics>true</noUsageStatistics>
+            </owner>
+            <name>Pipeline (Services Blue Green Deployment)</name>
+            <description>description</description>
+            <filterExecutors>false</filterExecutors>
+            <filterQueue>false</filterQueue>
+            <properties class="hudson.model.View$PropertyList"/>
+            <gridBuilder class="au.com.centrumsystems.hudson.plugin.buildpipeline.DownstreamProjectGridBuilder">
+              <firstJob>first-job</firstJob>
+            </gridBuilder>
+            <noOfDisplayedBuilds>10</noOfDisplayedBuilds>
+            <buildViewTitle>build title</buildViewTitle>
+            <consoleOutputLinkStyle>Lightbox</consoleOutputLinkStyle>
+            <cssUrl></cssUrl>
+            <triggerOnlyLatestJob>false</triggerOnlyLatestJob>
+            <alwaysAllowManualTrigger>false</alwaysAllowManualTrigger>
+            <showPipelineParameters>true</showPipelineParameters>
+            <showPipelineParametersInHeaders>false</showPipelineParametersInHeaders>
+            <startsWithParameters>true</startsWithParameters>
+            <refreshFrequency>3</refreshFrequency>
+            <showPipelineDefinitionHeader>true</showPipelineDefinitionHeader>
+          </outer-class>
+        </au.com.centrumsystems.hudson.plugin.buildpipeline.BuildPipelineView_-MyUserIdCause>
+      </causes>
+    </hudson.model.CauseAction>
+    <hudson.plugins.parameterizedtrigger.CapturedEnvironmentAction plugin="parameterized-trigger@2.17">
+      <env serialization="custom">
+        <unserializable-parents/>
+        <tree-map>
+          <default>
+            <comparator class="hudson.util.CaseInsensitiveComparator" reference="../../../../../org.jenkinsci.plugins.envinject.EnvInjectPluginAction/build/actions/hudson.plugins.parameterizedtrigger.CapturedEnvironmentAction/env/tree-map/default/comparator"/>
```

Comment:
@jglick maybe you have any idea how restore old installations with this reference?
int and string becomes exposed to Action and Action casting fails